x-pack/platform/plugins/shared/alerting/server/routes/alert_delete/apis/last_run/get_alert_delete_last_run_route.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { httpServiceMock } from '@kbn/core/server/mocks';
+import { licenseStateMock } from '../../../../lib/license_state.mock';
+import { alertDeleteLastRunRoute } from './get_alert_delete_last_run_route';
+import { mockHandlerArguments } from '../../../_mock_handler_arguments';
+import { alertDeletionClientMock } from '../../../../alert_deletion/alert_deletion_client.mock';
+import { rulesClientMock } from '../../../../rules_client.mock';
+
+jest.mock('../../../../lib/license_api_access', () => ({
+  verifyApiAccess: jest.fn(),
+}));
+
+jest.mock('../../../lib/track_legacy_terminology', () => ({
+  trackLegacyTerminology: jest.fn(),
+}));
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+const alertDeletionClient = alertDeletionClientMock.create();
+const rulesClient = rulesClientMock.create();
+
+describe('alertDeletePreviewRoute', () => {
+  it('registers the route without public access', async () => {
+    const licenseState = licenseStateMock.create();
+    const router = httpServiceMock.createRouter();
+
+    alertDeleteLastRunRoute(router, licenseState);
+    expect(router.get).toHaveBeenCalledWith(
+      expect.not.objectContaining({
+        options: expect.objectContaining({ access: 'public' }),
+      }),
+      expect.any(Function)
+    );
+    expect(router.get).toHaveBeenCalledWith(
+      expect.objectContaining({
+        options: expect.objectContaining({ access: 'internal' }),
+      }),
+      expect.any(Function)
+    );
+  });
+
+  it('gets the amount of affected alerts', async () => {
```

Comment:
I think this name is a copy paste that was not updated 😅

Reply:
done in 88ab0839d051552261b00b2e3bd9e10bdcbccafa